frodo-runtime/src/main/java/com/fernandocejas/frodo/internal/DebugLog.java

```diff
@@ -18,6 +21,28 @@
    * @param message The message you would like logged.
    */
   public void log(String tag, String message) {
-    Log.d(tag, message);
+    if (!runningJUnit())
```

Comment:
I tried to catch the exception but I think this kind of exception is not possible to catch.